src/libostree/ostree-repo-finder.h

```diff
@@ -130,6 +141,7 @@ typedef struct
   OstreeRepoFinder *finder;
   gint priority;
   GHashTable *ref_to_checksum;
+  GHashTable *ref_to_timestamp;
```

Comment:
I wonder if I need to decrement the padding variable to avoid ABI break. I think so